src/plugins/template/tests/functional/shared_tests_instances/behavior/plugin/preprocessing.cpp

```diff
@@ -4,8 +4,6 @@
 
 #include "behavior/plugin/preprocessing.hpp"
 
-#ifdef ENABLE_GAPI_PREPROCESSING
-
```

Comment:
let's drop

Reply:
File is removed

Reply:
I reverted deletion to unblock merge because openvino_contrib and npu tests depend on it